frontend/stores/dashboard/useUserDashboardStore.ts

```diff
@@ -118,6 +119,10 @@ export function useUserDashboardStore () {
 
   // Update the hash (=hashed list of id's) of a specific local dashboard
   function updateHash (type: DashboardType, hash: string) {
+    if (!isPublicDashboardKey(hash) || isSharedKey(hash)) {
+      warn('updateHash: invalid public hashed key: ', hash)
```

Comment:
nitpick: call stack should appear in `vue warn` anyway: but on the other hand I get that it is more convenient to read
```suggestion
      warn('invalid public hashed key: ', hash)
```

Reply:
sure ;-)